src/app/pages/office/cat-manoeuvre/office.cat-manoeuvre.page.html

```diff
@@ -45,6 +45,16 @@ <h4>Test summary</h4>
                             (activityCodeChange)="activityCodeChanged($event)"
                             [disabled]="!isDelegated">
                     </activity-code>
+                    <accompaniment-card [formGroup]="form"
```

Comment:
Can we move this down the just above the "Additional information" component as per Cat B please